spec/unit/telemeter_spec.rb

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright:: Copyright (c) 2018 Chef Software Inc.
+# Copyright:: Copyright (c) 2018-2019 Chef Software Inc.
```

Comment:
maybe some consistency? either a range or the year in both places? 🤔

Reply:
Fixed to include span in both places